ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-next__411.yaml

```diff
@@ -421,10 +457,6 @@ tests:
         name: knative-eventing-kafka-broker-test-wathola-sender
       - env: KNATIVE_EVENTING_KAFKA_BROKER_SOURCE_IMAGE
         name: knative-eventing-kafka-broker-source-image
-      - env: KNATIVE_EVENTING_KAFKA_BROKER_DISPATCHER
-        name: knative-eventing-kafka-broker-dispatcher
-      - env: KNATIVE_EVENTING_KAFKA_BROKER_RECEIVER
-        name: knative-eventing-kafka-broker-receiver
```

Comment:
/hold

Reply:
I'm not sure why we've this deletion and what was changed in hack recently

Reply:
I see here https://github.com/openshift-knative/hack/pull/84, we need to add a pattern